tests/scripts/test_stdlib_call.py

```diff
@@ -29,14 +31,14 @@ def test_stdin_fd():
     os.close(r)
     assert ret['stdout'] == '<LOREM IPSUM>\n'
 
-# FIXME: (**LINE_BUFFER**)
-# def test_linebuffer_callback():
-#     buffered = []
-#
-#     def callback(fd, data):
-#         buffered.append(data)
-#     _call(('bash', '-c', 'echo 1; echo 2; echo 3'), callback_linebuffered=callback)
-#     assert buffered == ['1', '2', '3']
+
+def test_linebuffer_callback():
+    buffered = []
+
+    def callback(fd, data):
```

Comment:
just nitpick, fd_info would be better, but I am ok with fd as well as it is not used.